index.js

```diff
@@ -53,3 +72,11 @@ export function moveFileSync(sourcePath, destinationPath, {overwrite = true, dir
 		}
 	}
 }
+
+export async function renameFile(source, destination, options) {
+	return moveFile(source, destination, options);
```

Comment:
Should it validate that the source and destination are in the same directory?

Reply:
I guess it depends on if `renameFile` is an alias to `moveFile` or if it only renames a file within a directory. Based on the name, the latter probably makes more sense.

Reply:
Yeah, lets validate it.